scripts/kotlin-lint-check

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
This file should end with .sh since it's a shell file.

Reply:
done